### PR TITLE
Feature - Product environment impact

### PR DESCRIPTION
**Pull request by maiconburn**
Add basic structure for the product environment impact page.

### CODE REVIEW

File: product-reader/product/index.html
Comment (saslam1023) [inline product-reader/product/index.html:1]: can we link to the main css in css/styles.css? to keep consistent with our design.

Comment (maiconburn) [reply product-reader/product/index.html:1]: Yeah sure, I will do it.

**Review by saslam1023 — CHANGES_REQUESTED** [review]
for consistency in the design we have used, can we link to the main css/styles.css ?

**Review by maiconburn — COMMENTED** [review]
Done

**Review by saslam1023 — APPROVED** [review]
Good to go then, Maicon!

**Pull request merged**